doc/01_intro.md

```diff
@@ -0,0 +1,37 @@
+# Introduction {#mainpage} #
+The purpose of a [synchronization client](@ref synchronization_client) for an AsteroidOS watch is to provide a means by which data and commands from the machine, generally, a computer or mobile phone running Linux on which the client runs, to the watch.  Although either a computer or mobile phone are commonly used for this, the rest of this document will refer to "the computer" to mean the computer on which the synchronization client runs.
+
+It's important to understand that this current work only provides *half* of the required software, and that a separate synchronization client then uses D-Bus to actually perform the synchronization.
+
+Data can include things like the time of day and weather data for the next few days, while commands can include things like taking a screenshot or turning the music volume up or down.
+
+Note that some things flow *from* the computer to the watch.  An example of this is weather forecast data which would be sent *from* the computer to the watch.  Some things flow *to* the computer from the watch, such as music player commands to turn the volume up or down or skip to the next track.  
+
+For all of these, the communications between the watch and the computer uses [Bluetooth](https://www.bluetooth.com/learn-about-bluetooth/tech-overview/).  Later pages describe the details of this and how it works. 
+
+## An example
+
+An example may be useful to clarify.  In this example, we trace a single request to turn the volume up via the watch's music app.  From the user's point of view, here's a very simplified sequence diagram.  This assumes that the sofware is already running on the computer, that the watch is [paired](@ref paired), and that the watch is [connected](@ref connected).  The music player shown here isn't any particular software, but a class of software that can play media.  Real examples of such software include [vlc](https://www.videolan.org/) and [Rhythmbox](https://wiki.gnome.org/Apps/Rhythmbox), but as we will see later, as long as the software implements a particular kind of interface, it can be used.
+
+\startuml
```

Comment:
Very cool! Didn't know this was even possible 🚀

Reply:
It is.  I have tested both!